code-of-conduct.md

```diff
@@ -0,0 +1,157 @@
+# <h1 align="center"> satRday Code of conduct/ Anti-Harassment policy </h1>
+
+
+
+
+
+## Short Version
+
+
+satRday is dedicated to a harassment-free and enjoyable conference experience for all participants.
+
+Our code of conduct/anti-harassment policy can be found at [here](http://capetown2018.satrdays.org/code-of-conduct.md).
```

Comment:
I get that the short and medium versions should link to the main code of conduct when they are referenced elsewhere, but it seems a little weird in this context since the markdown document contains a link to itself.

Could we remove this line?

If we want to retain this, it might be better to include this as a note at the top of the document:

>Any use of the short or medium versions of the code of conduct shown below should include a link back to this page as shown below.
>
>Our full code of conduct/anti-harassment policy can be found [here](http://capetown2018.satrdays.org/code-of-conduct.md).

Alternatively, and possibly slightly better - we could split this into two separate documents; an html page containing the long version, and a markdown document for the short and medium versions?